packages/internet-header/src/components/post-internet-footer/components/post-footer-block-contact.component.tsx

```diff
@@ -44,8 +42,8 @@ export const PostFooterBlockContact = (props: {
               {content.text ? <p class="text">{content.text}</p> : null}
               {content.hours && isLiveSupport && <LiveSupport hours={content.hours} />}
               {content.hours && !isLiveSupport && (
-                // Some values arrive in the form of <p>8&emdash;12</p> and without replace and innerHTML, tags get rendered as text (project="klp" language="en" environment="int02")
-                <p class="hours" innerHTML={getContentHours(content.hours)}></p>
+                // Some values arrive in the form of <p>8&mdash;12</p> and without replace and innerHTML, tags get rendered as text (project="klp" language="en" environment="int02")
```

Comment:
&emdash; is not decoded with any library. It seems to never have existed (at least officially), see: https://bugzilla.mozilla.org/show_bug.cgi?id=221606